src/project/qml/MuseScore/Project/AlsoShareAudioComDialog.qml

```diff
@@ -0,0 +1,132 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0-only
+ * MuseScore-CLA-applies
+ *
+ * MuseScore
+ * Music Composition & Notation
+ *
+ * Copyright (C) 2023 MuseScore BVBA and others
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+
+StyledDialogView {
+    id: root
+
+    property string audioComUrl
+    property bool rememberChoice
+
+    contentHeight: 355
+    contentWidth: 496
+
+    margins: 24
+
+    function done(data = {}) {
+        let value = Object.assign(data)
+
+        root.ret = {
+            errcode: 0,
+            value: value
+        }
+
+        root.hide()
+    }
+
+    ColumnLayout {
+        id: content
+
+        anchors.fill: parent
+        spacing: 20
+
+        ColumnLayout {
+            id: header
+
+            width: parent.width
+            spacing: 16
+
+            StyledTextLabel {
+                Layout.fillWidth: true
+
+                //: The text between `<a href=\"%1\">` and `</a>` will be a clickable link to Audio.com
+                text: qsTrc("project/cloud", "Would you also like to share your music on <a href=\"%1\">Audio.com</a>?").arg(root.audioComUrl)
```

Comment:
We need to voice the entire dialog text. See

https://github.com/musescore/MuseScore/blob/5f57ce04e0978db91f6195a2ea6f92a4030a4b5b/src/framework/ui/qml/MuseScore/Ui/StandardDialogPanel.qml#L126

Also 

https://github.com/musescore/MuseScore/blob/5f57ce04e0978db91f6195a2ea6f92a4030a4b5b/src/framework/ui/qml/MuseScore/Ui/StandardDialogPanel.qml#L225